File: 이지수/Swift/Baekjoon_1929.swift
import Foundation
func solution(_ m: Int, _ n: Int) {
    for num in m...n where isPrime(num) { print(num) }
}
func isPrime(_ num: Int) -> Bool {
    guard num > 2 else { return num == 2 }
    for i in 2...Int(sqrt(Double(num)))+1 where num % i == 0 { return false }
    return true
}
let input = readLine()!.split(separator: " ").map(){Int($0)!}
solution(input[0], input[1])

File: 이지수/Python/Baekjoon_1929.py
import sys
import math
input = sys.stdin.readline

def isPrime(num) :
    if num < 3 : return num == 2
    for i in range(2, math.ceil(math.sqrt(num))+1) :
        if num % i == 0 : return False 
    return True

m, n = map(int, input().split())
for num in range(m, n+1) :
    if isPrime(num) : print(num)
